<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormRunSequence</class>
 <widget class="QWidget" name="FormRunSequence">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>211</width>
    <height>113</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>211</width>
    <height>113</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>211</width>
    <height>113</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Running Sequence...</string>
  </property>
  <widget class="QLabel" name="label1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Actual Step:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Next Step:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label2_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Time left:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonPause">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Pause </string>
   </property>
   <property name="icon">
    <iconset resource="tripod_lamp_gui.qrc">
     <normaloff>:/Tripod_Lamp_GUI/Icons/control_pause_blue.png</normaloff>:/Tripod_Lamp_GUI/Icons/control_pause_blue.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonStop">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>80</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop </string>
   </property>
   <property name="icon">
    <iconset resource="tripod_lamp_gui.qrc">
     <normaloff>:/Tripod_Lamp_GUI/Icons/control_stop_blue.png</normaloff>:/Tripod_Lamp_GUI/Icons/control_stop_blue.png</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="labelStepNow">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelStepNext">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>30</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelTime">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>50</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="tripod_lamp_gui.qrc"/>
 </resources>
 <connections/>
</ui>
